pkg/subctl/benchmark/latency.go

```diff
@@ -0,0 +1,56 @@
+package benchmark
+
+import (
+	"fmt"
+
+	"github.com/onsi/gomega"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func StartLatencyTests() {
+	gomega.RegisterFailHandler(func(message string, callerSkip ...int) { panic(message) })
```

Comment:
Shouldn't we run `cleanupFramework(f) ` on assertion failure?

Reply:
Oh right, Let's move this line after L13, and call cleanupFramework before the panic.

Reply:
the RegisterFailHandler() need to be called before framework init

Reply:
the RegisterFailHandler() need to be called before framework init

Reply:
You could use a variable for the framework

`var f framework.Framework`  before the gomega.RegisterFailHandler


Then 

`f = initFramework("latency")`